db/migrate/20161104150651_add_broken_link_count_to_local_authority.rb

```diff
@@ -0,0 +1,11 @@
+class AddBrokenLinkCountToLocalAuthority < ActiveRecord::Migration[5.0]
+  def up
+    add_column :local_authorities, :broken_link_count, :integer, default: 0
+
+    LocalAuthority.all.map(&:calculate_count_of_broken_links)
```

Comment:
Quick question, is this doing the same as `LocalAuthority.all.each { |la| la.calculate_count_of_broken_links }`? Does using the `map` method have any advantages here?